app.js

```diff
@@ -1,26 +1,64 @@
 const express = require("express");
-const logger = require("morgan");
-const cors = require("cors");
-
-const contactsRouter = require("./routes/api/contacts");
+const connectDB = require("./db");
+const {
+  getAllContacts,
+  addContact,
+  updateContact,
+  deleteContact,
+} = require("./controlers/contacts");
 
 const app = express();
 
-const formatsLogger = app.get("env") === "development" ? "dev" : "short";
+connectDB();
 
-app.use(logger(formatsLogger));
-app.use(cors());
 app.use(express.json());
 
-app.use("/api/contacts", contactsRouter);
+app.get("/api/contacts", async (req, res) => {
+  try {
+    const contacts = await getAllContacts();
```

Comment:
Od teraz tu zawsze podawać musisz owner
```suggestion
    const contacts = await getAllContacts(owner: req.user._id);
```
nie jestem pewien czy dobrze napisałem, ale chodzi o to teraz to musi być bo dany user ma tylko swoje kontakty.